ta/secure_key_services/src/processing_ec.c

```diff
@@ -1239,13 +1239,14 @@ uint32_t generate_ec_keys(struct sks_attribute_head *proc_params,
 			  struct sks_attrs_head **priv_head)
 {
 	uint32_t rv = 0;
-	void *a_ptr = NULL;
-	uint32_t a_size = 0;
+	void *a_ptr = NULL, *b_ptr = NULL;
+	uint32_t a_size = 0, b_size = 0;
```

Comment:
In optee, we tend to use 1 line per local var. 
That not a strong opinion but helps.

Reply:
sure.